src/GLTypes.jl

```diff
@@ -220,8 +225,17 @@ function GLVertexArray(bufferdict::Dict, program::GLProgram)
             )
             bind(buffer)
             attribLocation = get_attribute_location(program.id, attribute)
+            glslType = attributeTypes[Symbol(attribute)]
             (attribLocation == -1) && continue
-            glVertexAttribPointer(attribLocation, cardinality(buffer), julia2glenum(eltype(buffer)), GL_FALSE, 0, C_NULL)
+            if glslType in GLSLFloatTypes
+                glVertexAttribPointer(attribLocation, cardinality(buffer), julia2glenum(eltype(buffer)), GL_FALSE, 0, C_NULL)
```

Comment:
While we're at it, we might want to correctly infer the normalization part of `glVertexAttribPointer`, which should be `GL_TRUE` for fixedpoint numbers!